apps/studio/components/interfaces/Auth/Policies/AIPolicyEditorPanel/index.tsx

```diff
@@ -407,28 +409,50 @@ export const AIPolicyEditorPanel = memo(function ({
                       disabled={isExecuting || incomingChange !== undefined}
                       onClick={() => onExecuteSQL()}
                     >
-                      Save policy
+                      Create policy
```

Comment:
I reckon save is better? This UI is also applicable for editing an existing policy

Reply:
I'm not going to die on this hill, but my thoughts are:

You can't save a policy, it's either inserted or created.
Because the code you write is not "saved", it's "run" against the db

Reply:
i'd have to disagree here haha 😅 
yes the code you write is run, but the consequence of the code run is "saving" a policy -> which is more important from the user POV imo since this is the outcome

Reply:
```suggestion
                      Save policy
```

Reply:
Ok reverting..

need to be careful with copy in other areas, we keep swinging from create/save terminology.

<img width="641" alt="Screenshot 2024-02-26 at 5 28 35 PM" src="https://github.com/supabase/supabase/assets/8291514/be6224b6-5dac-4303-959c-67acdaf835bb">

Reply:
got it. yea no worries.